source/_components/media_player.firetv.markdown

```diff
@@ -27,59 +25,61 @@ Steps to configure your Amazon Fire TV stick with Home Assistant:
 - Find Amazon Fire TV device IP:
   - From the main (Launcher) screen, select Settings.
   - Select System > About > Network.
-- The following commands must be run in a Python 2.x environment. They will allow the component to function in an Ubuntu 16.04/Hassbian environment.
-  - `apt-get install swig libssl-dev python-dev libusb-1.0-0 python-yaml`
-  - `pip install flask`
-  - `pip install https://pypi.python.org/packages/source/M/M2Crypto/M2Crypto-0.24.0.tar.gz`
-  - `pip install firetv[firetv-server]`
-- `firetv-server -d <fire tv device IP>:5555`, background the process
-- Navigate to http://localhost:5556/devices/list
-  - You will get an output similar to below:
-```json
-{
-  "devices": {
-    "default": {
-      "host": "192.168.1.153:5555", 
-      "state": "play"
-    }
-  }
-}
-```
-- The `"default"` above is the device name you will need to use for your `configuration.yaml` 
-- Configure Home Assistant as follows:
 
 To add FireTV to your installation, Note your device name, and add the following to your `configuration.yaml` file:
 
 ```yaml
 # Example configuration.yaml entry
 media_player:
+  # a device that does not require ADB authentication
   - platform: firetv
+    name: Fire TV 1
+    host: 192.168.0.111
+
+  # a device that does require ADB authentication
+  - platform: firetv
+    name: Fire TV 2
+    host: 192.168.0.222
+    adbkey: "/config/android/adbkey"
+
+  # a device for which getting the current app (source) and the running apps (sources) cause issues
+  - platform: firetv
+    name: Fire TV 3
+    host: 192.168.0.123
+    get_source: false
+    get_sources: false
 ```
 
-{% configuration %}
-host:
-  description: "The host where `firetv-server` is running."
-  required: false
-  type: string
-  default: localhost
-port:
-  description: "The port where `firetv-server` is running."
-  required: false
-  type: integer
-  default: 5556
-device:
-  description: The device ID.
-  required: false
-  type: string
-  default: default
-name:
-  description: The friendly name of the device.
-  required: false
-  type: string
-  default: Amazon Fire TV
-{% endconfiguration %}
-
-<p class='note warning'>
-Note that python-firetv has support for multiple Amazon Fire TV devices. If you have more than one configured, be sure to specify the device ID in `device`. Run `firetv-server -h` and/or view the source for complete capabilities.
-</p>
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration